src/main/java/embl/ebi/variation/commons/models/metadata/FileGenerator.java

```diff
@@ -39,7 +34,13 @@
     private static final long serialVersionUID = -5926609525556333330L;
     
     protected String alias;
-    @Transient protected Set<File> files = new HashSet<>();
+    @ManyToMany(cascade = CascadeType.PERSIST)
+    @JoinTable(
+            name="FILE_GENERATOR_FILE",
```

Comment:
I think the defaults for Spring JPA are in lower-case (please check), so it could be better to have something like "filegenerator_file" as join table name.